src/index.ts

```diff
@@ -67,3 +118,13 @@ const jhubAppsPlugin: JupyterFrontEndPlugin<void> = {
 const plugins = [jhubAppsPlugin];
 
 export default plugins;
+
+class deployAppWidget extends Widget {
+  constructor() {
+    super();
+    this.addClass('jp-example-view');
+    this.id = 'deploy-app';
+    this.title.label = 'Deploy App';
+    this.title.closable = true;
```

Comment:
I think we would want to have an icon in the tab bar, we can reuse the same rocket icon for now.